MANIFEST.in

```diff
@@ -20,6 +20,15 @@ recursive-include docs *.yml
 recursive-include docs *.png
 recursive-include docs Makefile
 recursive-include tests *
+prune tests/__pycache__
```

Comment:
A lot of these changes should be in a separate pull request?

Reply:
https://github.com/SymbiFlow/fasm/pull/33